input/pagecontent/StructureDefinition-ncpi-sample-intro.md

```diff
@@ -1,8 +1,7 @@
 ### Basic Information
-Collected samples represent biological samples derived from a given participant with information about the 
 
 #### Key Guidelines
-The NCPI Sample profile is based on the standard resource type, [Specimen](https://hl7.org/fhir/R4/specimen.html) and encompasses biospecimen collection, sample information, and aliquot information. The NCPI Collected Sample is based on the [NCPI Sample](StructureDefinition-ncpi-sample) Profile. The main difference between a collected and non-collected sample involves inclusion or exclusion of container information for a sample as "NCPI Sample" is comprised of Sample, Aliquot, and Biospecimen Collection.
+The NCPI Sample profile is based on the standard resource type, [Specimen](https://hl7.org/fhir/R4/specimen.html) and encompasses biospecimen collection, sample information, and aliquot information. Biospecimen collection information can be excluded if unavailable. 
 
```

Comment:
I don't know who should write it (and maybe it will go in a different file), but we should probably mention the gotcha regarding parent sample and collection (requiring one or the other).